src/python/engagement-creator/src/engagement-creator.py

```diff
@@ -294,6 +312,10 @@ def _process_one_event(
         create_edge(mg_client, node.uid, "risks", risk.uid)
         create_edge(mg_client, risk.uid, "risky_nodes", node.uid)
 
+        # Or perhaps we should just emit per-risk instead of per-risky-node?
+        # this path is here for now just to appease the soc2 gods; will improve usability later
```

Comment:
idt internal soc2 details need to be in the public repo

Reply:
agreed